app/views/start_page.scala.html

```diff
@@ -20,35 +20,75 @@
-                </p>
-                <p><a href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/customs-international-trade-and-excise-enquiries" target="_blank" rel="noopener noreferrer">@messages("startPage.enquiries")</a></p>
-            </div>
-        </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      @components.heading(
+        sectionHeader = messages("startPage.title.sectionHeader"),
+        title = messages("startPage.title")
+      )
+    </div>
+    <div class="govuk-grid-column-two-thirds">
+      <p id="description" class="lede">@messages("startPage.description")</p>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+
+    <p id="contents">@messages("startPage.contents.header")</p>
+    <ul class="dashed">
+      <li id="contents-element-1"><a href="#before-you-start">@messages("startPage.beforeYouStart.header")</a></li>
+      <li id="contents-element-2"><a href="#information-you-need">@messages("startPage.informationYouNeed.header")</a></li>
+      <li id="contents-element-3"><a href="#report-your-arrival-and-departure">@messages("startPage.reportYourArrivalAndDeparture.header")</a></li>
+    </ul>
+
+    <h2 id="before-you-start" class="heading-medium">@messages("startPage.beforeYouStart.header")</h2>
+    <p id="before-you-start-line-1">
+      @messages("startPage.beforeYouStart.line.1") <a href=@customsDeclarationsOutOfEuUrl>@messages("startPage.beforeYouStart.line.1.link")</a>
+    </p>
+    <p id="before-you-start-line-2">@messages("startPage.beforeYouStart.line.2")</p>
+    <p id="before-you-start-line-3">@messages("startPage.beforeYouStart.line.3")</p>
+
+    <h2 id="information-you-need" class="heading-medium">@messages("startPage.informationYouNeed.header")</h2>
+    <p id="information-you-need-line-1">@messages("startPage.informationYouNeed.line.1")</p>
+    <ul class="list list-bullet">
+      <li id="information-you-need-listItem-1">@messages("startPage.informationYouNeed.listItem.1")</li>
+      <li id="information-you-need-listItem-2">@messages("startPage.informationYouNeed.listItem.2")</li>
+      <li id="information-you-need-listItem-3">@messages("startPage.informationYouNeed.listItem.3")</li>
+      <li id="information-you-need-listItem-4">@messages("startPage.informationYouNeed.listItem.4")</li>
+    </ul>
+
+    <h2 id="report-your-arrival-and-departure" class="heading-medium">@messages("startPage.reportYourArrivalAndDeparture.header")</h2>
+
+    <div class="panel panel-border-wide">
+      <p id="problems-with-service-notice">
+        @messages("startPage.problemsWithServiceNotice") <a href=@serviceAvailabilityUrl>@messages("startPage.problemsWithServiceNotice.link")</a>
+      </p>
     </div>
+
+    <br/>
+    <a id="button-start" class="button button-start" href="@routes.ChoiceController.displayChoiceForm" role="button">
+      @messages("startPage.buttonName")
+    </a>
+
+    <div>
+      <br/>
```

Comment:
Do we have to get those 4 brs? Can we have 4 em top margin?

Reply:
Good point. Changed.